docs/src/main/asciidoc/redis-reference.adoc

```diff
@@ -656,6 +656,81 @@ The reactive variant returns a `Uni<Response>`.
 
 NOTE: You can also execute custom command in a transaction.
 
+== Preload data into Redis
+
+On startup, you can configure the Redis client to preload data into the Redis database.
+
+=== Load scripts
+
+Specify the _load script_ you want to load using:
+
+[source, properties]
+----
+quarkus.redis.load-script=import.sql # import.sql is the default in dev, no-file is the default in prod
+quarkus.redis.my-redis.load-script=actors.redis, movies.redis
+----
+
+IMPORTANT: `load-script` is a build time property than cannot be overridden at runtime.
+
+Note that each client can have a different script, even a list of scripts.
+In the case of a list, the data is imported in the list order (for example, first `actors.redis`, then `movies.redis` for the `my-redis` client).
+
+=== Load Script format
+
+The `.redis` file follows a _one command per line_ format:
```

Comment:
I'll be awesome if Quarkus developer tools can support for this.
Who do we tag here? @cescoffier